src/App.css

```diff
@@ -33,14 +33,21 @@
 .Nav a {
   text-decoration: none;
 }
+.section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 160px 0 0 0;/*needed to dodge header and nav */
+  border: 2px solid blue;
+
+}
 .ArticleList {
   list-style: none;
   display: grid;
   grid-template-columns:  1fr; /*only does one column at below first min-width below*/
   grid-gap:0.25rem;
   max-width: 1400px;
   padding: 0.25rem;
-  margin: 160px 0 0 0;
   border: 2px solid orange;
 }
 @media (min-width: 350px) { /* renders 2 columns above this width  */
```

Comment:
For what it's worth, media queries tend to be for bigger viewports and mobile-first for your regular CSS :)